index.js

```diff
@@ -247,7 +247,7 @@ module.exports = {
 			'always',
 			{
 				// You can also ignore this rule by wrapping the first word in quotes.
-				ignorePattern: /pragma|ignore|prettier-ignore/.source,
+				ignorePattern: /pragma|ignore|prettier-ignore|webpack/.source,
```

Comment:
```suggestion
				ignorePattern: /pragma|ignore|prettier-ignore|webpack\w+:/.source,
```

I assume all the magic comments end in `:`?

I just want to ensure that comment like:

```js
// webpack does not like this
```

are caught.